bigquery_etl/glam/templates/histogram_bucket_counts_v1.sql

```diff
@@ -17,33 +13,83 @@ build_ids AS (
     app_build_id,
     channel,
   FROM
-    all_combos
+    filtered_data
   GROUP BY
     1,
     2
   HAVING
-      COUNT(DISTINCT client_id) > {{ minimum_client_count }}),
-histograms_cte AS (
-  SELECT
-    {% if channel == "release" %}
-      sampled,
+    {% if use_sample_id %}
+      -- At least <min_client_count>*sample_size clients in the sample
+      COUNT(DISTINCT client_id) > {{ minimum_client_count }} * (@max_sample_id - @min_sample_id + 1)/100
+    {% else %}
+      COUNT(DISTINCT client_id) > {{ minimum_client_count }}
     {% endif %}
-    {{ attributes }},
-    ARRAY(
-      SELECT AS STRUCT
-        {{metric_attributes}},
-        {% if channel == "release" %}
-        -- Logic to count clients based on sampled windows release data, which started in v119.
-        -- If you're changing this, then you'll also need to change
-        -- clients_daily_[scalar | histogram]_aggregates
-          mozfun.glam.histogram_normalized_sum_with_original(value, IF(sampled, 10.0, 1.0)) AS aggregates,
-        {% else %}
-          mozfun.glam.histogram_normalized_sum_with_original(value, 1.0) AS aggregates,
-        {% endif %}
-      FROM unnest(histogram_aggregates)
-    )AS histogram_aggregates
+),
+data_with_enough_wau AS (
```

Comment:
wau isn't the right measurement right?  It's just clients per build id?
```suggestion
data_with_enough_clients AS (
```